security/pfSense-pkg-acme/files/usr/local/pkg/acme.xml

```diff
@@ -28,10 +28,10 @@
 	]]>
 	</copyright>
 	<name>acme</name>
-	<title>Acme</title>
+	<title>ACME</title>
 	<include_file>/usr/local/pkg/acme/acme.inc</include_file>
 	<menu>
-		<name>Acme Certificates</name>
+		<name>ACME Certificates</name>
```

Comment:
Changing the menu name is fairly disruptive. Even just changing the case, it won't match in the code that adds/removes menu entries and some users will end up with two menu entries depending on what they do.